django_fido/admin/authenticator.py

```diff
@@ -28,6 +33,7 @@ class Fido2RegistrationRequestAdminView(PermissionRequiredMixin, Fido2Registrati
     """Registration request view with user selection."""
 
     permission_required = 'django_fido.add_authenticator'
+    user_verification = UserVerificationRequirement.DISCOURAGED
```

Comment:
I'd add a comment that this is a workaround for Chrome throwing an error even for a correct PIN.

Reply:
I totaly redid the code, so this is no longer relevant.